pandoc_drawio_filter/pandoc_drawio_filter.py

```diff
@@ -27,8 +28,15 @@ def drawio(key, value, format_, _):
         if src_extension == ".drawio":
             pdf_name = f"{src_basename}.pdf"
             if modification_time(pdf_name) < modification_time(src):
+                # Set the drawio binary based on the operating system
+                if platform.system() == "Darwin":
+                    drawio_binary = "/Applications/draw.io.app/Contents/MacOS/draw.io"
```

Comment:
I see that this one aims towards making the script more portable, but unluckily there are cases where it gets actually less portable this way.

When you run `nix-shell -p pandoc pandoc-drawio-filter`, `drawio` will be already in the path of the `pandoc-drawio-filter` script. This works on linux and on MacOS. With this change, the package would have to patch it back from `/Applications/draw.io.app/....` to the `drawio` call.

A variant that first tries to find `drawio` and if that can't be found, tries the `/Applications/...` path as the next resort?